shared/src/ledger/protocol/transactions/ethereum_events/eth_msgs.rs

```diff
@@ -31,7 +31,7 @@ impl From<MultiSignedEthEvent> for EthMsgUpdate {
     ) -> Self {
         Self {
             body: event,
-            seen_by: signers.into_iter().collect(),
```

Comment:
This was already deterministically deduping though taking the latest vote rather than the earliest one.